demos/crossroad_camera_demo/README.md

```diff
@@ -73,6 +74,12 @@ For example, to do inference on a GPU with the OpenVINO&trade; toolkit pre-train
 ./crossroad_camera_demo -i <path_to_video>/inputVideo.mp4 -m <path_to_model>/person-vehicle-bike-detection-crossroad-0078.xml -m_pa <path_to_model>/person-attributes-recognition-crossroad-0230.xml -m_reid <path_to_model>/person-reidentification-retail-0079.xml -d GPU
 ```
 
+> **NOTE**: The detection network returns as the result a set of detected objects, where each detected object consists of a bounding box and an index of the object's category (person/vehicle/bike). The demo runs Person Attributes Recognition and Person Reidentification networks only for the bounding boxes that has the category "person".
+> Since different detection networks may have different category index corresponding to the category "person", this index may be pointed by the command line parameter `-person_label`.
+> Please, note that
+> * the model `person-vehicle-bike-detection-crossroad-0078` returns for persons the category index 1, it is the default value for the demo
+> * the model `person-vehicle-bike-detection-crossroad-1016` returns for persons the category index 2, so for the correct work of the demo the command line parameter `-person_label 2` should be added.
```

Comment:
It would be useful to have something similar to that note in that model description

Reply:
Ok, I will add it.

Reply:
Done